### PR TITLE
Add Carbon Graphs to /open_source

### PR DESCRIPTION
**Pull request by amcginn**
For reference: https://engineering.cerner.com/open_source/
Preview:
FireShot Capture 004 - Engineering Health - localhost

### CODE REVIEW

File: source/open_source/index.html [hunk 1]
```diff
@@ -56,6 +58,17 @@ <h2>Canadarm</h2>
...
       </div>
+      <div class="row-fluid">
+        <div class="span4">
+          <h2>Carbon Graphs</h2>
+          <img src="/open_source/carbon-graphs.png" alt="GC Stats" />
+          <p>
+            Carbon is a graphing library built using D3 visualization library. It provides an API
+            for generating native graphs such as Line and Paired Result graphs based on design
+            standards.
+            <a href="https://github.com/cerner/gc_stats">Learn More...</a>
```
Comment (RBecho) [inline source/open_source/index.html]: url is https://github.com/cerner/carbon-graphs

File: source/open_source/index.html [hunk 2]
```diff
@@ -56,6 +58,17 @@ <h2>Canadarm</h2>
           </p>
         </div>
       </div>
+      <div class="row-fluid">
+        <div class="span4">
+          <h2>Carbon Graphs</h2>
+          <img src="/open_source/carbon-graphs.png" alt="GC Stats" />
```
Comment (RBecho) [inline source/open_source/index.html]: Alt should be Carbon Graphs

File: source/open_source/index.html [hunk 3]
```diff
@@ -41,7 +41,9 @@ <h2>Beadledom</h2>
           <h2>Bunsen</h2>
```
Comment (RBecho) [inline source/open_source/index.html]: Bunsen and Canadarm appear to be missing row-fluid div's. I wonder if that's why the vertical alignment isn't a 100% perfect vertically aligned grid?

Comment (RBecho) [reply source/open_source/index.html]: or maybe that should / shouldn't be on every element? I'll admit my div / css knowledge is sub zero, but it seems like the alignment vertically of the OSS software page images is a little wonky

Comment (amcginn) [reply source/open_source/index.html]: good catch. missed that I copied an extra div. fixed:
FireShot Capture 005 - Engineering Health - localhost

**Pull request merged**